mesheryctl/internal/cli/root/system/start.go

```diff
@@ -49,8 +52,300 @@ import (
 var (
 	skipUpdateFlag  bool
 	skipBrowserFlag bool
+	configureCluster string
 )
 
+func getContexts(configFile string) ([]string, error) {
+	mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+	if err != nil {
+		utils.Log.Error(err)
+		return nil, nil
```

Comment:
```suggestion
		return nil, err
```
error should be handled from the caller